conferences.json

```diff
@@ -210,5 +210,18 @@
             "hackaton": "yes",
             "live streaming": "yes"
         }
-    }
+    },
+    {
+        "name": "RustConf",
+        "date": "2020-08-20T09:30:00-07:00",
+        "blurb": "RustConf 2020 online",
+        "location": "Portland, USA",
```

Comment:
Just figured that it should've said online.

Reply:
Ah! Yes, that is correct, it is all online.